FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/ISchedulerService.cs

```diff
@@ -16,6 +16,12 @@ public interface ISchedulerService
         /// </summary>
         public DateTimeOffset LastHeartbeat { get; set; }
 
+        public int SchedulerServicePullingIntervalInSeconds { get; set; }
+
+        public int SchedulerServiceLeaseExpirationInSeconds { get; set; }
```

Comment:
Where those be used?

Reply:
The default values are used now. But I think they should be configurable, and the caller can configure them in `JobManager` as `jobHosting`

Reply:
shall we reference these fields in the caller method? Otherwise they will be forgot in the future